### ISSUE TITLE
Please refrain from reselling Icehacks under a different brand name

### ISSUE THREAD

**Issue by StabbyAlbert** [epic-hacks/epic-hacks-surviv.io#12]
Icehacks GP has taken notice of this github repository. Please do not sell our cheat or you may suffer from legal repercussions. 
[https://youtu.be/7LG9bW1TJKQ
](https://youtu.be/7LG9bW1TJKQ
)

**Comment by humphreygaming** [comment]
Taken from terms and conditions:

**TERMS AND CONDITIONS**
**These are the terms for all my products.**

Use of this product require the obeying of these written rules.
- reproduction of this product with claims to creation is not allowed
- duplication is not allowed
- production in videos is prohibited unless you have permission
- this product is not for retail sale or any other sale
- I am not responsible for the outcomes of your actions

basically this is mine and I am keeping it that way.

**Comment by epic-hacks** [comment]
who said that we are using your cheat in ours? seems like that discussion is happening in [that](https://github.com/epic-hacks/epic-hacks-surviv.io/issues/10) issue
